File: exercises-python/2. Problemas de estructura condicional/11.py
#11.Desarrolle el programa que determine el signo de un número entre positivo, negativo y cero.

import os
os.system("cls")

#Datos
num = int(input("Número: "))

#Proceso
if num > 0:
    print("Es positivo")
elif num < 0:
    print("Es negativo")
else:
    print("Es cero.")

File: exercises-cpp/2. Condicionales/11.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
using namespace std;

int main()
{
    system("cls");
    
    int num;

    printf("Número: "); cin>>num;

    if (num > 0){
        printf("Es positivo");}
    else if (num < 0){
        printf("Es negativo");}
    else{
        printf("Es cero.");}
}